e2e.py

```diff
@@ -0,0 +1,74 @@
+"""Simple Bigtable client demonstrating auto-generated veneer.
+"""
+
+import argparse
+import logging
+
+from google.bigtable.v1 import bigtable_service_api
+from google.bigtable.admin.cluster.v1 import bigtable_cluster_service_api
+from google.bigtable.admin.table.v1 import bigtable_table_service_api
+
+from google.bigtable.v1 import bigtable_data_pb2 as data
+from google.bigtable.admin.cluster.v1 import bigtable_cluster_data_pb2 as cluster_data
+from google.bigtable.admin.table.v1 import bigtable_table_data_pb2 as table_data
+
+def run(project_id):
+  with bigtable_service_api.BigtableServiceApi() as bigtable_api, \
```

Comment:
Discussed with Sai that we should refactor the example to have a single function inside the with clause. @geigerj.